python/paddle/fluid/dygraph/parallel.py

```diff
@@ -284,6 +286,8 @@ def scale_loss(loss):
 def apply_collective_grads(parameters):
     if not ParallelEnv().world_size > 1:
         return
+    warnings.warn("If you use DataParallel, "
```

Comment:
Can we delete this function if we don’t need it?

Reply:
ok, i will remove it .